pkg/controllers/openshift-controller-manager.go

```diff
@@ -50,7 +50,7 @@ func NewOpenShiftControllerManager(cfg *config.MicroshiftConfig) *OCPControllerM
 
 func (s *OCPControllerManager) Name() string { return componentOCM }
 func (s *OCPControllerManager) Dependencies() []string {
-	return []string{"kube-apiserver", "openshift-apiserver"}
+	return []string{"kube-apiserver"}
```

Comment:
Would/should this depend on the openshift-crd-manager instead, to wait until the CRDs become available?

If so, you'd need to swap the order of adding these services in `run.go`, too.

Reply:
Ordering is still fraught, but I've added this in another commit.